src/main/groovy/com/entagen/jenkins/JenkinsJobManager.groovy

```diff
@@ -64,7 +65,9 @@ class JenkinsJobManager {
         for(ConcreteJob missingJob in missingJobs) {
             println "Creating missing job: ${missingJob.jobName} from ${missingJob.templateJob.jobName}"
             jenkinsApi.cloneJobForBranch(missingJob, templateJobs)
-            jenkinsApi.enableJob(missingJob.jobName)
+            if (!isUnitTestOnly && !missingJob.jobName.contains("Rev.com-featurebranch-deploy-feature")) {
```

Comment:
Thinking some more about this `Rev.com-featurebranch` is the prefix. But if we are sharing job we should change this to `Rev.com-branch` and the last feature in `Rev.com-featurebranch-deploy-feature` is actually the name of the branch so this should really search for `Rev.com-branch-deploy-` and the next one should be ``Rev.com-branch-build-`